service/templates/admin.html

```diff
@@ -2546,6 +2546,9 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 									</md-button>
 								</md-card-title>
 								<md-card-content>
+										<div class="small" style="height:25px;">
```

Comment:
Is it needed to have a feature flag for this since it's a route? Also presumably it should error if the thing isn't "enabled" rather than just return nothing.

Reply:
Since this isn't a route that users can manually call, and has to specify something in their task yaml, I think it makes sense. 

We wouldn't want this feature not being enabled to affect(effect?) the success of a task so I thought it made sense.

Reply:
But < 10 isn't an intuitive check. Could we just make it 0? Like, if the feature is _actually off_ then bail.

Reply:
yeah I initially made it 10 because I was playing around with different numbers on staging to test other things, but we can keep it 0 on prod and something else on staging. I'll change it to 0.